style/style.scss

```diff
@@ -0,0 +1,67 @@
+$colortitulos: red;
+$colorsubtitulos: firebrick;
+$colornombres: brown;
+$background: cornflowerblue;
+$fuente: 20px ;
+$fuentefamilia: "Times New Roman", Times, serif;
+$imagenes: 270px;
```

Comment:
Always use english in variables and code

Reply:
Also, always use hexadecimal colors instead of named ones

`red` => `#FF0000` and so on